### PR TITLE
Make OCP version more configurable

### PR DESCRIPTION
**Pull request by dmc5179**
Current vars file has about 13 different places where the version number needs to be changed to move to another OCP version. Update vars file to shorten this to 2 variables. I've tested the playbooks and everything downloads just the same.

### CODE REVIEW

**Comment by christianh814** [comment]
> Current vars file has about 13 different places where the version number needs to be changed to move to another OCP version. Update vars file to shorten this to 2 variables. I've tested the playbooks and everything downloads just the same.

The Z stream of RHCOS doesn't always match the Z stream of openshift

meaning, for example, the following scenario is possible.

OpenShift Latest Version: 4.3.15
RHCOS Latest Version: 4.3.8

The only reason it works in your case is because `4.3.8` is an openshift version AND the rhcos version. This will break if you try to install the latest, which is `4.3.18`

**Comment by dmc5179** [comment]
hmm, well it was totally hard coded before and everything pegged to 4.3.8 so it certainly doesn't seem like supporting mixed versioning due to the Z stream not matching was a goal. I'll update the code to allow for the split.

**Comment by christianh814** [comment]
@dmc5179 Can you change this PR to the `devel` branch?

**Comment by christianh814** [comment]
There's been 3 releases since this PR and it's gone stale. Feel free to submit another PR (to the `devel` branch) if you'd still like to see this change
